activerecord/test/cases/adapters/postgresql/enum_test.rb

```diff
@@ -111,6 +111,16 @@ def test_schema_dump
     assert_includes output, 't.enum "good_mood", default: "happy", null: false, enum_type: "mood"'
   end
 
+  def test_schema_dump_with_dot_in_enum_name
```

Comment:
Dots are an easy way to demonstrate the behavior, but quote identifiers are also important for case sensitivity.